ComponentKit/Components/CKImageComponentAutoSized.h

```diff
@@ -0,0 +1,26 @@
+/*
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+#import <ComponentKit/CKCompositeComponent.h>
+
+struct CKComponentSize;
+
+/**
+ A component that displays an image using UIImageView.
+ */
+@interface CKImageComponentAutoSized : CKCompositeComponent
```

Comment:
This violates the naming convention we have. CKAutoSizedImageComponent would be appropriate.